### PR TITLE
LITE-24594 building fixed

### PR DESCRIPTION
**Pull request by zzzevaka**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -15,6 +15,19 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
```
Comment (maxipavlovic) [inline .github/workflows/build.yml]: I would suggest to leave only 3.8+ here in checks.

File: tests/reports/out.xml [hunk 2]
```diff
@@ -1 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="0" skipped="0" tests="6" time="0.977" timestamp="2022-07-27T09:03:59.457913" hostname="22ce2a36705d"><testcase classname="tests.test_backend" name="test_reconnect_enabled" time="0.440" /><testcase classname="tests.test_backend" name="test_reconnect_disabled" time="0.020" /><testcase classname="tests.test_backend" name="test_reconnect_in_atomic_before_begin[False]" time="0.047" /><testcase classname="tests.test_backend" name="test_reconnect_in_atomic_before_begin[True]" time="0.045" /><testcase classname="tests.test_backend" name="test_not_reconnect_in_transaction[False]" time="0.021" /><testcase classname="tests.test_backend" name="test_not_reconnect_in_transaction[True]" time="0.063" /></testsuite></testsuites>
```
Comment (maxipavlovic) [inline tests/reports/out.xml:1]: Shouldn't this be ignored?

**Review by vgrebenschikov — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -11,6 +11,19 @@ jobs:
     strategy:
```
Comment (maxipavlovic) [inline .github/workflows/deploy.yml:11]: Shouldn't this file be called publish?

**Review by maxipavlovic — APPROVED** [review]
LGTM

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (zzzevaka) [reply .github/workflows/deploy.yml:11]: It was stolen from lib-rql)

image

File: tests/reports/out.xml (hunk 2 above)
Comment (zzzevaka) [reply tests/reports/out.xml:1]: Should. I deleted this in this PR and added to gitignore

File: pyproject.toml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 name = "django-postgresql-reconnect"
 homepage = "https://connect.cloud.im"
```
Comment (marcserrat) [inline pyproject.toml]: I will point to https://github.com/cloudblue/django-postgresql-reconnect

File: pyproject.toml [hunk 5]
```diff
@@ -2,7 +2,7 @@
...
 repository = "https://github.com/cloudblue/django-postgresql-reconnect"
-version = "0.0.1"
+version = "0.0.0"
 description = ""
 authors = ["Django PostgreSQL database wrapper"]
```
Comment (marcserrat) [inline pyproject.toml]: MM......maybe you?

File: pyproject.toml [hunk 6]
```diff
@@ -19,9 +19,9 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "3.10.*"
-Django = "^4.0.6"
-psycopg2 = "^2.9.3"
+python = "^3.6"
```
Comment (marcserrat) [inline pyproject.toml]: Will force python >= 3.8

File: pyproject.toml [hunk 7]
```diff
@@ -19,9 +19,9 @@ classifiers = [
...
+Django = "^3.2"
```
Comment (marcserrat) [inline pyproject.toml:21]: maybe in future also add django testing matrix as we have with CQRS?

**Review by marcserrat — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (zzzevaka) [reply .github/workflows/build.yml]: done

File: pyproject.toml (hunk 4 above)
Comment (zzzevaka) [reply pyproject.toml]: done

File: pyproject.toml (hunk 5 above)
Comment (zzzevaka) [reply pyproject.toml]: Cloudblue LLC :) done

File: pyproject.toml (hunk 6 above)
Comment (zzzevaka) [reply pyproject.toml]: done

File: pyproject.toml (hunk 7 above)
Comment (zzzevaka) [reply pyproject.toml:21]: will keep it for the next version if you don't mind

**Review by d3rky — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**